packages/asl/package.json

```
@@ -12,7 +12,7 @@
     "test": "npm run test:lint && npm run test:deduped",
     "test:deduped": "./test/dedupe.sh",
     "test:lint": "eslint . --ext jsx,js",
-    "test:audit": "audit-ci --high --skip-dev --config .auditrc.json",
+    "test:audit": "audit-ci --config .auditrc.json",
```

Unless these args are breaking something we should stick to the setup used prior to merging into the mono repo.

```suggestion
    "test:audit": "audit-ci --high --skip-dev --config .auditrc.json",
```